exercises/leap/introduction.md

```diff
@@ -0,0 +1,13 @@
+# Introduction
+
+A leap year (in the Gregorian calendar) occurs in every year that is evenly divisible by 4, except if the year is evenly divisible by 100, unless the year is also evenly divisible by 400.
```

Comment:
This sentence is a bit tricky to parse with the except-unless clause. Do those both apply to the base rule? Does the unless apply to except? Does except and unless mean something different?
I'm wondering if it makes sense to split this into multiple sentences and make the exceptions a bit more explicit.

Reply:
Maybe like:

> A leap year (in the Gregorian calendar) occurs for every year evenly divisible by 4, but there are two exceptions.
If the year is evenly divisible by 100 it is not considered a leap year, unless it is also evenly divisible by 400.

Reply:
How about:

```suggestion
A leap year (in the Gregorian calendar) occurs:
- In every year that is evenly divisible by 4
- Unless the year is evenly divisible by 100, in which case it's only a leap year if the year is also evenly divisible by 400.
```

Reply:
That looks good to me!